cmd/to-html.go

```diff
@@ -0,0 +1,534 @@
+	}
+
+	// sort results by level
+	// order: error, warning, note, none, unknown
+	sort.Slice(results, func(i, j int) bool {
+		return levelOrder[results[i].Properties["Level"].(string)] < levelOrder[results[j].Properties["Level"].(string)]
+	})
+}
+
+func calculateMD5Hash(text string) string {
+	hash := md5.New()
+	io.WriteString(hash, text)
+	return hex.EncodeToString(hash.Sum(nil))
+}
+
+// function that calculates a fingerprint for threadflow
+func calculateThreadFlowFingerprint(threadFlow *sarif.ThreadFlow) string {
+	var fingerprint string
+	for _, location := range threadFlow.Locations {
+		fingerprint += fmt.Sprintf("|%s:%d:%d:%d:%d;",
+			location.Location.PhysicalLocation.ArtifactLocation.Properties["URI"].(string),
+			location.Location.PhysicalLocation.Region.Properties["StartLine"].(int),
+			location.Location.PhysicalLocation.Region.Properties["StartColumn"].(int),
+			location.Location.PhysicalLocation.Region.Properties["EndLine"].(int),
+			location.Location.PhysicalLocation.Region.Properties["EndColumn"].(int),
+		)
+	}
+	return calculateMD5Hash(fingerprint)
+}
+
+// remove codeflow duplicates
+// each codeflow may have multiple threatflows. These threatflows may be equal for different codeflows.
+// This function removes duplicates from codeflows
+// if the codeflow is empty, it is removed
+func removeDataflowDuplicates(results []*sarif.Result) {
+	for _, result := range results {
+		uniqueThreadFlowsFingerprints := map[string]bool{}
+		for _, codeFlow := range result.CodeFlows {
+			uniqueThreadFlows := []*sarif.ThreadFlow{}
+			for _, threadFlow := range codeFlow.ThreadFlows {
+				fingerprint := calculateThreadFlowFingerprint(threadFlow)
+				if _, ok := uniqueThreadFlowsFingerprints[fingerprint]; !ok {
+					uniqueThreadFlowsFingerprints[fingerprint] = true
+					uniqueThreadFlows = append(uniqueThreadFlows, threadFlow)
+				}
+			}
+			codeFlow.ThreadFlows = uniqueThreadFlows
+		}
+
+		// remove empty codeflows
+		nonEmptyCodeFlows := []*sarif.CodeFlow{}
+		for _, codeFlow := range result.CodeFlows {
+			if len(codeFlow.ThreadFlows) > 0 {
+				nonEmptyCodeFlows = append(nonEmptyCodeFlows, codeFlow)
+			}
+		}
+		result.CodeFlows = nonEmptyCodeFlows
+	}
+}
+
+// function that collects information about amount of low, mediumn and high severity issues
+// returns a map with this information, and a total amount of issues
+func collectSeverityInfo(sarifReport *sarif.Report) map[string]int {
+	severityInfo := map[string]int{
+		"low":    0,
+		"medium": 0,
+		"high":   0,
+		"total":  0,
+	}
+
+	for _, run := range sarifReport.Runs {
+		for _, result := range run.Results {
+			severity := result.Properties["Level"].(string)
+			switch severity {
+			case "error":
+				severityInfo["high"]++
+			case "warning":
+				severityInfo["medium"]++
+			default:
+				severityInfo["low"]++
+			}
+			severityInfo["total"]++
+		}
+	}
+
+	return severityInfo
+}
+
+// toHtmlCmd represents the toHtml command
+var toHtmlCmd = &cobra.Command{
```

Comment:
Can you a description for the help command:

https://github.com/scan-io-git/scan-io/blob/main/cmd/analyse.go#L28
It's not the refactored example, but better than nothing